applications/hub/targets/linux.rpi1/AvaHiMDNS.cxx

```diff
@@ -0,0 +1,140 @@
+/** \copyright
```

Comment:
This file looks like an exact copy of the one under targets/linux.x86.  Please make this a symbolic link instead.

Reply:
Instead, it would be better to move this code to use os/MDNS.hxx/cxx I think since this code looks almost verbatim for both the linux.x86 and now linux.rpi1. I was contemplating making this update since I'm using this same hub app on my rPi 3B+ but the linux.x86 app compiled and works on there without alteration (I ran make from top level dir)

Reply:
Yes, I had to do a copy/paste of the code to get a clean compile. Making this a shared file would be good